templates/database.ini.erb

```diff
@@ -15,3 +15,6 @@ password = <%= scope.lookupvar('puppetdb::db_password') %>
 
 # How often (in minutes) to compact the database
 # gc-interval = 60
+<%- [@options].flatten.each do |singleopt| -%>
+<%= singleopt %>
```

Comment:
Are you sure this brings the expected output? I expect options to be an hash and for each key of that hash have a line here.
I'd do something like:
```
<% @options.each do |key,value| -%>
<%= key %> = <%= value %>
<% end -%>
```